agent/engine/docker_task_engine.go

```diff
@@ -1100,7 +1100,7 @@ func getFirelensLogConfig(task *apitask.Task, container *apicontainer.Container,
 }
 
 func (engine *DockerTaskEngine) startContainer(task *apitask.Task, container *apicontainer.Container) dockerapi.DockerContainerMetadata {
-	seelog.Infof("Task engine [%s]: starting container: %s", task.Arn, container.Name)
+	seelog.Infof("Task engine [%s]: starting container: %s (Docker ID: %s)", task.Arn, container.Name, container.RuntimeID)
```

Comment:
how about calling it Runtime ID?

Reply:
How about Container ID? Doesnt `docker ps` call it Container ID?

Reply:
> how about calling it Runtime ID?

 Should be a quick find/replace if you agree with either sharanyad or sparrc.  My vote’s for “Runtime ID” (since it’s `container.RuntimeID`)

Reply:
but this is an external-facing log message. Docker doesnt externally call it "Runtime ID" anywhere. For example googling "docker runtime id" produces cryptic results. Googling "docker container id" produces exactly what we're talking about.

Reply:
In ECS, we have been calling it Runtime ID - https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Container.html#ECS-Type-Container-runtimeId

Reply:
Thanks for the suggestions. Changed it to "Runtime ID" in order to be consistent with ECS public documentation.